CMake/AbseilDll.cmake

```diff
@@ -499,12 +505,14 @@ set(ABSL_INTERNAL_DLL_TARGETS
   "log_initialize"
   "log"
   "log_entry"
+  "log_flags"
```

Comment:
Sorry for the delay. The comment does apply to log_flags as well. If you remove this, test it, and it works, I will merge this.

Reply:
Done. We've been shipping this patch for several months in conda-forge, so I'm pretty confident that it works. :)

Reply:
Have you built and run the tests under MSVC? This doesn't appear to build.

Reply:
We built it successfully as the PR looked until yesterday, i.e. _with_ `log_flags` as a dll target. I can start a new build and see if it now fails with `log_flags` removed

Reply:
So I've tried building this on top of 20230125.0 in https://github.com/conda-forge/abseil-cpp-feedstock/pull/56, and it works (or perhaps: "works"?) like before. Not sure if anything else relevant happened on the LTS branch, if necessary, I can check that too...

In any case, I cannot see the failure being shown by the CI below, what's the error?

PS. For context again (since this PR is old): the idea is to produce as few static libraries as possible when doing `BUILD_SHARED_LIBS=ON` on windows; `flags_*` are an exception, but the other I'm adding here should be more oversights than fundamentally-needs-to-be-static libs.